docs/DeveloperGuide.adoc

```diff
@@ -333,34 +402,66 @@ Use case ends.
 * 2a. The list is empty.
 +
 Use case ends.
-
++
+[none]
 * 3a. The given index is invalid.
 +
 [none]
 ** 3a1. AddressBook shows an error message.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Add event
+
+*MSS*
+
+1.  User keys in an event
+2.  TeaPet adds the event to the schedule
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. Event is invalid.
++
+[none]
+** 2a1. TeaPet shows an error message.
++
+Use case ends.
+
 _{More to be added}_
 
 [appendix]
 == Non Functional Requirements
 
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
 .  Should be able to hold up to 1000 students without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  A teacher with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  TeaPet should be used only for a teacher handling his/her own form class.
```

Comment:
The NFRs are well-written but perhaps you could improve its definition by quantifying the number of tasks so that it will be a clearer requirement to meet.